.gitignore

```diff
@@ -4,3 +4,4 @@ dist/**/*
 **/*.egg-info/**/*
 .sass-cache/*
 /build
+/_readthedocs/
```

Comment:
Nope.

Reply:
Removed in https://github.com/RadeonOpenCompute/rocm-docs-core/pull/140/commits/9ad2c7f3d6cbfe368372416fa2d19b5c1644d8c9